movie.sh

```diff
@@ -1,12 +1,18 @@
 #!bin/bash
+
 function getrating()
 {
-curl "http://www.omdbapi.com/?t=$1&y=&plot=short&r=xml" > mov.txt
+#Fetch tuple from the Omdb API
+curl -s  "http://www.omdbapi.com/?t=$1&y=&plot=short&r=xml" > mov.txt
```

Comment:
You should leave files on the file system only when they are going to be used after the execution of your program. If there is no other way to proceed without writing a file, those files must be written at the `/tmp` directory. Read more about `/tmp` here: http://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/tmp.html